pkg/logs/sds/reconfigure.go

```diff
@@ -6,21 +6,73 @@
 //nolint:revive
 package sds
 
+import (
+	pkgconfigmodel "github.com/DataDog/datadog-agent/pkg/config/model"
+)
+
 type ReconfigureOrderType string
 
+const WaitForConfigField = "logs_config.sds.wait_for_configuration"
+const WaitForConfigBufferMaxSizeField = "logs_config.sds.buffer_max_size"
+const WaitForConfigDefaultBufferMaxSize = 10000
```

Comment:
At 256K max message size, this is 2,5Gb? Would it make sense to limit number of bytes stored, rather than the number of log messages?

Reply:
I have to think about this : this buffering mechanism is in place only for a persona that might not even exist (people who will start using SDS + using socket/syslog listeners and/or who can't use the `no_collection` mechanism), it's only here to deal with the seconds it take to reach the RC intake at startup (or an eventual unreachable RC _at startup_).

It's an undocumented field as I don't expect any user to have to adapt it, and to be simpler to think about in terms of count of logs for people who'll have to, but I might be wrong, I'll consider this.

Reply:
Switched to a limit in bytes.